feat: add disable options to SelectArrayInput (#5940)
marmelab/react-admin: docs/Inputs.md

];
```

> You can set disabled values by setting the `disabled` property of one item: 

Reviewer: ```suggestion
You can render any item as disabled by setting its `disabled` property to `true`: 
```

```jsx
const choices = [
    { _id: 123, full_name: 'Leo Tolstoi', sex: 'M' },
    { _id: 456, full_name: 'Jane Austen', sex: 'F' },
    { _id: 1, full_name: 'System Administrator', sex: 'F', disabled: true },
];
<SelectArrayInput source="author_id" choices={choices} optionText="full_name" optionValue="_id" />
```

> You can use a custom field name by setting `disableValue` prop: 

Reviewer: ```suggestion
You can use a custom field name by setting the `disableValue` prop: 
```
